packages/gatsby-plugin-netlify-cms/package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "gatsby-plugin-netlify-cms",
   "description": "A Gatsby plugin which generates the Netlify CMS single page app",
-  "version": "2.0.0-beta.3",
+  "version": "2.2.0-beta.15",
```

Comment:
Can you revert this change? Versioning will be handled during the release process.